SF iOS/SF iOS/Location/FeedFetch/FeedFetchService.m

```diff
@@ -22,8 +22,8 @@ - (instancetype)init {
     return self;
 }
 
--(void)getFeedWithHandler:(FeedFetchCompletionHandler)completionHandler {
-    FeedFetchOperation *operation = [[FeedFetchOperation alloc] initWithCompletionHandler:^(NSArray<NSDictionary *> *feed, NSError *_Nullable error) {
+-(void)getFeedAtURLString:(NSString*)endpoint withHandler:(FeedFetchCompletionHandler)completionHandler {
```

Comment:
There should be a space after `NSString` and before the `*` to be consistent with the rest of the project.